README.md

```diff
@@ -174,12 +175,13 @@ A valid configuration file should look like this:
 
 The following table describes the configuration properties defined above.
 
-| Setting                 | Description                        | Default        |
-|-------------------------|------------------------------------|----------------|
-| bridge.id               | ID of the bridge                   | my-bridge      |
-| mqtt.host               | Host address of the MQTT server    | localhost      |
-| mqtt.port               | Port number of the MQTT server     | 1883           |
-| kafka.bootstrap.servers | Bootstrap servers for Apache Kafka | localhost:9092 |
+| Setting                        | Description                                                                                      | Default          |
+|--------------------------------|--------------------------------------------------------------------------------------------------|------------------|
+| bridge.id                      | ID of the bridge                                                                                 | my-bridge        |
+| bridge.topic.default | Default Kafka topic for the bridge. If not set, the bridge will use the `messages_default` topic | messages_default |
```

Comment:
The column separator is not aligned with the rest.
Also it should state something like "Topic to be used if no matches with any mapping rules".
Finally, no need to add "If not set, the bridge will use the `messages_default` topic" because it's clear from the Default column.

Reply:
That makes sense. 
In this order, I think we are not handling default bridge ID. We are expecting the user to enter the `.properties` file. 
If the user does not set it, the `bridge-id` will be `null`.

Reply:
@ppatierno could you take a look at this?

Reply:
AFAICS the bridge.id is not even actually used so the null value doesn't have any problem.
Also, compared to HTTP bridge we use the bridge.id for generating names to producer and consumer endpoints (something we don't have) and if bridge.id is null we just generate a UUID for those names. But again the bridge.id itself is not used by itself. I think for now we are fine.

Reply:
Yeah, it is fine!